kano_updater/scenarios.py

```diff
@@ -359,4 +359,22 @@ def beta_210_to_beta_220(self):
             )
 
     def beta_220_to_beta_230(self):
-        pass
+        os.system('sudo adduser {} i2c'.format(get_user_unsudoed()))
```

Comment:
@radujipa @convolu Shoudln't this be os.system('sudo usermod -a -G i2c {}'.format(get_user_unsudoed()))? Adduser adds an actual user.